docs/index.asciidoc

```diff
@@ -462,6 +463,15 @@ The size of the TCP send buffer (SO_SNDBUF) to use when sending data
 The timeout after which, if the `poll_timeout_ms` is not invoked, the consumer is marked dead
 and a rebalance operation is triggered for the group identified by `group_id`
 
+[id="plugins-{type}s-{plugin}-ssl_endpoint_identification_algorithm"]
+===== `ssl_endpoint_identification_algorithm`
+
+  * Value type is <<string,string>>
+  * Default value is `"https"`
+
+The endpoint identification algorithm, defaults to "https". Set to empty string "" to disable endpoint verification
+
```

Comment:
Please put `"https"` and `""` in backticks.  
Otherwise, builds cleanly and  LGTM. Thanks!

Reply:
Done - thank you @karenzone!